docs/sdk/fragments/_get-user-information.mdx

```diff
@@ -0,0 +1,36 @@
+import ScopeClaimList from './_scope-claim-list.md';
+
+{props.getIdTokenClaimsApi && props.fetchUserInfoApi &&
+
+<>
+You can use Logto API to retrieve user information:
+
+<ul>
+  <li>
+    <code>{props.getIdTokenClaimsApi}</code>: get user information from ID token.
```

Comment:
getIdTokenClaims is not an API? Also might double check all the SDKs provides that method

Reply:
Also just like sign-in sign-out, should we provide sample code as props here? 
calling getIdTokenClaimsn and getUserInfo using different SDK.

Reply:
Update. Only add to those docs where related apis are available.